demo/fake_gan/run.py

```diff
@@ -40,12 +39,12 @@ def fake_gan(count, *args):
     title="FD-GAN",
     description="This is a fake demo of a GAN. In reality, the images are randomly chosen from Unsplash.",
     examples=[
-        [2, cheetah, 12, None, None, None],
-        [1, cheetah, 2, None, None, None],
-        [4, cheetah, 42, None, None, None],
-        [5, cheetah, 23, None, None, None],
-        [4, cheetah, 11, None, None, None],
-        [3, cheetah, 1, None, None, None],
+        [2, cheetah, None, 12, None, None],
```

Comment:
Maybe this broke fake_gan?

![fake_gan_error](https://user-images.githubusercontent.com/41651716/185415057-9c05d5c7-0705-49aa-9464-cd238d49f554.gif)

Reply:
Let me take a look

Reply:
Good catch! Pushed a fix